packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-options-tile/OrganizationOptionsTile.component.tsx

```diff
@@ -12,9 +12,7 @@ export default function OrganizationOptionsTile() {
       <div className="flex flex-col w-full">
```

Comment:
There is a component in the lib for this : DashboardTile
You should maybe use it to not have to update it later with the ODS versions